README.rst

```diff
@@ -40,6 +40,35 @@ The AMD Inference Server is integrated with the following libraries out of the g
 * XModel models with `Vitis AI <https://www.xilinx.com/products/design-tools/vitis/vitis-ai.html>`__ on AMD FPGAs
 * A graph of computation including as pre- and post-processing can be written using `AKS <https://github.com/Xilinx/Vitis-AI/tree/v2.5/src/AKS>`__ on AMD FPGAs for end-to-end inference
 
+Quick Start Inference
+----------------------------------------
+
+Quick start requires AMD EPYC CPU::
```

Comment:
This example does not require EPYC CPUs to run. Performance will be different but that doesn't matter in this example.

Reply:
The ZenDNN runtime works on any CPU but may be specially tuned for performance on EPYCs

Reply:
done.